src/LogTarget.php

```diff
@@ -152,53 +203,4 @@ protected function gc(&$manifest)
             }
         }
     }
-
-    /**
-     * Collects summary data of current request.
-     * @return array
-     */
-    protected function collectSummary()
-    {
-        if (Yii::$app === null) {
-            return '';
-        }
-
-        $request = Yii::$app->getRequest();
-        $response = Yii::$app->getResponse();
-        $summary = [
-            'tag' => $this->tag,
-            'url' => $request->getAbsoluteUrl(),
-            'ajax' => (int) $request->getIsAjax(),
-            'method' => $request->getMethod(),
-            'ip' => $request->getUserIP(),
-            'time' => $_SERVER['REQUEST_TIME_FLOAT'],
-            'statusCode' => $response->statusCode,
-            'sqlCount' => $this->getSqlTotalCount(),
-        ];
-
-        if (isset($this->module->panels['mail'])) {
-            $mailFiles = $this->module->panels['mail']->getMessagesFileName();
-            $summary['mailCount'] = count($mailFiles);
-            $summary['mailFiles'] = $mailFiles;
-        }
-
-        return $summary;
-    }
-
-    /**
-     * Returns total sql count executed in current request. If database panel is not configured
-     * returns 0.
-     * @return int
-     */
-    protected function getSqlTotalCount()
-    {
-        if (!isset($this->module->panels['db'])) {
-            return 0;
-        }
-        $profileLogs = $this->module->panels['db']->getProfileLogs();
-
-        # / 2 because messages are in couple (begin/end)
-
-        return count($profileLogs) / 2;
-    }
 }
```

Comment:
why did you change order of methods here? this makes the whole pull request very hard to review... :(

Reply:
Yes, definitely...

Reply:
I'm sorry. I undid the order change of methods in a separate branch:
[master...without-reformat](https://github.com/yiisoft/yii2-debug/compare/master...simialbi:without-reformat)

Reply:
https://github.com/yiisoft/yii2-debug/pull/341